signer/migrations/0013__consolidate_withdrawal_tables.sql

```diff
@@ -0,0 +1,10 @@
+
+DROP TABLE sbtc_signer.withdrawal_create_events;
```

Comment:
We probably shouldn't drop this quite yet. There is a risk that someone has constructed a withdrawal request and has their funds locked up and we have a row for it in our database. It's very low probability, but we can check to make sure and remove it in any follow-up release.